x11-misc/screenkey/screenkey-1.3.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+DISTUTILS_USE_SETUPTOOLS=bdepend
+
+inherit distutils-r1 xdg
+
+DESCRIPTION="A screencast tool to display your keys inspired by Screenflick"
+HOMEPAGE="https://www.thregr.org/~wavexx/software/screenkey/"
+
+if [[ "${PV}" == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.com/screenkey/${PN}.git"
+else
+	SRC_URI="https://gitlab.com/screenkey/${PN}/-/archive/v${PV}/${PN}-v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}"/"${PN}"-v"${PV}"
+fi
+
+RESTRICT="test"
+LICENSE="GPL-3"
+SLOT="0"
+
+BDEPEND="
+	dev-python/python-distutils-extra[${PYTHON_USEDEP}]
+"
+RDEPEND="
+	dev-python/pycairo[${PYTHON_USEDEP}]
+	dev-python/pygobject[${PYTHON_USEDEP}]
+	sys-devel/gettext
+"
```

Comment:
There are some missing dependencies here like gtk+, slop, fontawesome and libappindicator.

Reply:
> libappindicator

or kde-plasma/xembed-sni-proxy in KDE cause I use KDE w/o dev-libs/libappindicator and have tray icon

Reply:
Yes, I think we can hide libappindicator behind USE=appindicator, only needed for GNOME shell.

Reply:
> Yes, I think we can hide libappindicator behind USE=appindicator, only needed for GNOME shell.

sure